pkg/fleet/installer/bootstrap/bootstrap_windows.go

```diff
@@ -75,18 +76,22 @@ func downloadInstaller(ctx context.Context, env *env.Env, url string, tmpDir str
 		return nil, fmt.Errorf("failed to extract layers: %w", err)
 	}
 
-	msis, err := filepath.Glob(filepath.Join(tmpDir, "datadog-agent-*-x86_64.msi"))
+	installPath, err = getInstallerFromMSI(tmpDir)
 	if err != nil {
-		return nil, err
-	}
-	if len(msis) > 1 {
-		return nil, fmt.Errorf("too many MSIs in package")
-	} else if len(msis) == 0 {
-		return nil, fmt.Errorf("no MSIs in package")
+		installPath, err = getInstallerFromOCI(tmpDir)
+		if err != nil {
+			return nil, fmt.Errorf("failed to get installer: %w", err)
+		}
```

Comment:
this hides the original error, would be nice to keep them both

```suggestion
        msiErr := fmt.Errorf("failed to find installer in MSI: %w", err)
		installPath, err = getInstallerFromOCI(tmpDir)
		if err != nil {
		    err = fmt.Errorf("failed to find installer in OCI: %w", err)
			return nil, fmt.Errorf("%w, %w", msiErr, err)
		}
```